tests/psa-client-server/psasim/test/Makefile

```diff
@@ -1,5 +1,16 @@
-INCLUDE := -I../include/ -I./psa_manifest
-LIB := -L../src -lpsaff
+LIBPSASIM_PATH := ..
+LIBPSACLIENT_PATH := ../../../libpsaclient
```

Comment:
could we put the root path e.g `../../..` into a variable so when the library moves it will be simpler?